zowe/vscode-extension-for-zowe
# feat(cfg): Allow extenders to contribute to Zowe schema during activation

## Proposed changes

This PR allows extenders to contribute new profile types to the Zowe schema during activation.

**How to test:**
- Install the Zowe Explorer FTP extension while the Zowe Explorer VSIX (from this PR) is installed
- After the FTP extension is activated, a new file should be created in the user's Zowe folder (`extenders.json`)
  - The contents of this file should contain the `zftp` profile type, with the `from` array containing `Zowe Explorer (for VS Code)`

NOTE: I recommend uninstalling the FTP extension and completely reloading VS Code before trying the steps above. VS Code will not always replace the contents of the extension folder if the same version is already installed.

## Release Notes

Milestone: 2.15.0

Changelog:

- Added the capability for extenders to contribute new profile types to the Zowe schema during extender activation. #2508 

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_


## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_
